library/core/src/ptr/mod.rs

```diff
@@ -1766,6 +1769,8 @@ pub unsafe fn read_volatile<T>(src: *const T) -> T {
 #[stable(feature = "volatile", since = "1.9.0")]
 #[rustc_diagnostic_item = "ptr_write_volatile"]
 #[cfg_attr(miri, track_caller)] // even without panics, this helps for Miri backtraces
+// What happens to the value that was replaced? No drop?
```

Comment:
What does that mean?

Reply:
Oops... that was a previous note. Let me delete it.

Reply:
```suggestion
```